src/services/refactors/extractMethod.ts

```diff
@@ -908,25 +921,30 @@ namespace ts.refactor.extractMethod {
         Write = 2
     }
 
-    export interface UsageEntry {
+    interface UsageEntry {
         readonly usage: Usage;
         readonly symbol: Symbol;
         readonly node: Node;
     }
 
     export interface ScopeUsages {
-        usages: Map<UsageEntry>;
-        typeParameterUsages: Map<TypeParameter>; // Key is type ID
-        substitutions: Map<Node>;
+        readonly usages: Map<UsageEntry>;

```

Comment:
What happens when a property of an exported type depends on a non-exported type?

Reply:
Users can't reference the type itself, but they can get a variable of the type. Pretty annoying. But it looks like this doesn't actually need to be exported.